.travis.yml

```diff
@@ -0,0 +1,24 @@
+sudo: false
+
+addons:
+    apt:
+        packages:
+            - doxygen
```

Comment:
actualy, docker-base travis only support 12.04. it related travis run speed.
i want to avoid version holding until found specific reason. it can make slow testing.
current released ubuntu still use v1.8.11
so if you want to hold v1.8.12, we need compile that version.

i only found one diff things, v1.7.x version not support markdown.(`support markdown` is main feature of v1.8.x)
IMO, it can acceptable. but i'll check another way.)